server/src/test/java/org/apache/uniffle/server/ShuffleServerMetricsTest.java

```diff
@@ -240,7 +242,9 @@ public void testNettyMetrics() throws Exception {
     ObjectMapper mapper = new ObjectMapper();
     JsonNode actualObj = mapper.readTree(content);
     assertEquals(2, actualObj.size());
-    assertEquals(68, actualObj.get("metrics").size());
+    // the original metrics have 69 items before this commit, but is would grow up
+    // when the new metric is added, we needn't correct this test while adding new metrics
+    assertTrue(actualObj.get("metrics").size() > 50);
```

Comment:
Prefer not. This is also a check for metrics.

Reply:
Ok, revert it back.